db/queries/assessment_records/queries.py

```diff
@@ -273,14 +274,13 @@ def get_application_jsonb_blob(application_id: str) -> dict:
     return application_json
 
 
-def update_status_to_completed(application_id):
-    from flask import current_app
-    current_app.logger.info(
-             f"Updating application status to COMPLETED")
+def update_status(application_id, status):
+    if status not in Status.__members__:
```

Comment:
Do we want to allow all statuses here? Previously we were just allowing `QA_COMPLETE` or `COMPLETED`?

Reply:
thought it could be worth allowing all statuses to future-proof this function/ make more flexible if needed?